app_mobilenet_alpha_1/src/main.xc

```diff
@@ -0,0 +1,76 @@
+#include "flash_server.h"
+#include "stdio.h"
+#include <platform.h>
+#include <quadflash.h>
+#include <stdint.h>
+
+#define NUMBER_OF_MODELS 1
+
+#if defined __GNUC__
+#define ALIGN(X) __attribute__((aligned(X)))
+#elif defined _MSC_VER
+#define ALIGN(X) __declspec(align(X))
+#elif defined __TASKING__
+#define ALIGN(X) __align(X)
+#endif
+
+#define FL_QUADDEVICE_MACRONIX_MX25R6435FM2IH0                                 \
+  {                                                                            \
+    16,                         /* MX25R6435FM2IH0 */                          \
+        256,                    /* page size */                                \
+        32768,                  /* num pages */                                \
+        3,                      /* address size */                             \
+        3,                      /* log2 clock divider */                       \
+        0x9F,                   /* QSPI_RDID */                                \
+        0,                      /* id dummy bytes */                           \
+        3,                      /* id size in bytes */                         \
+        0xC22817,               /* device id */                                \
+        0x20,                   /* QSPI_SE */                                  \
+        4096,                   /* Sector erase is always 4KB */               \
+        0x06,                   /* QSPI_WREN */                                \
+        0x04,                   /* QSPI_WRDI */                                \
+        PROT_TYPE_NONE,         /* no protection */                            \
+        {{0, 0}, {0x00, 0x00}}, /* QSPI_SP, QSPI_SU */                         \
+        0x02,                   /* QSPI_PP */                                  \
+        0xEB,                   /* QSPI_READ_FAST */                           \
+        1,                      /* 1 read dummy byte */                        \
+        SECTOR_LAYOUT_REGULAR,  /* mad sectors */                              \
+        {4096, {0, {0}}},       /* regular sector sizes */                     \
+        0x05,                   /* QSPI_RDSR */                                \
+        0x01,                   /* QSPI_WRSR */                                \
+        0x01,                   /* QSPI_WIP_BIT_MASK */                        \
+  }
+
+fl_QuadDeviceSpec flash_spec[] = {
+    FL_QUADDEVICE_MACRONIX_MX25R6435FM2IH0,
```

Comment:
Probably a question for @henkmuller. Should we include all flash spec types in here, instead of just the one? Or maybe better to not have them here at all.